### ISSUE TITLE
Error adding (1028, u'client-error-not-possible')

### ISSUE THREAD

**Issue by XaF** [simoncadman/CUPS-Cloud-Print#97]
Hello,

Contrary to the error presented in #96 my error alternates between `(1280, 'server-error-internal-error')` when using the .deb package and `(1028, u'client-error-not-possible')` when using git.

As you can see here using master:
```bash
xaf@duck:~/git/CUPS-Cloud-Print$ (git:master)> sudo ./setupcloudprint.py 
You currently have these accounts configured: 
xxxxxxxx@gmail.com
Error adding: GCP-printer1
(1028, u'client-error-not-possible')
Error adding: GCP-printer1
Error adding: GCP-printer2
(1028, u'client-error-not-possible')
Error adding: GCP-printer2 
Error adding: GCP-Enreg_dans_Drive
(1028, u'client-error-not-possible')
Error adding: GCP-Enreg_dans_Drive 
Error adding: GCP-printer3
(1028, u'client-error-not-possible')
Error adding: GCP-printer3 
Added 4 new printers to CUPS
```

And using `capabilities-fix`:
```bash
xaf@duck:~/git/CUPS-Cloud-Print$ (git:capabilities-fix)> sudo ./setupcloudprint.py 
You currently have these accounts configured: 
xxxxxxxx@gmail.com
Add more accounts (Y/N)? 
Add all Google Cloud Print printers from xxxxxxxx@gmail.com to CUPS (Y/N)? Y
Use a prefix for names of created printers (Y/N)? Y
Prefix ( e.g. GCP- )? GCP-
Error adding: GCP-printer1 (1028, u'client-error-not-possible')
Error adding: GCP-printer2 (1028, u'client-error-not-possible')
Error adding: GCP-Enreg_dans_Drive (1028, u'client-error-not-possible')
Error adding: GCP-printer3 (1028, u'client-error-not-possible')
Added 4 new printers to CUPS
```

In each cases, I previously ran the `upgrade.py` script.

In cups' `error_log`, I can see for the `(1280, 'server-error-internal-error')` case:
```
E [04/Apr/2016:12:08:24 -0400] copy_model: empty PPD file
E [04/Apr/2016:12:08:24 -0400] [Client 6] Returning IPP server-error-internal-error for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-printer1) from localhost
E [04/Apr/2016:12:08:24 -0400] copy_model: empty PPD file
E [04/Apr/2016:12:08:24 -0400] [Client 6] Returning IPP server-error-internal-error for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-printer2) from localhost
E [04/Apr/2016:12:08:24 -0400] copy_model: empty PPD file
E [04/Apr/2016:12:08:24 -0400] [Client 6] Returning IPP server-error-internal-error for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-Enreg%5Fdans%5FDrive) from localhost
E [04/Apr/2016:12:08:25 -0400] copy_model: empty PPD file
E [04/Apr/2016:12:08:25 -0400] [Client 6] Returning IPP server-error-internal-error for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-printer3) from localhost
```

And for the `(1028, u'client-error-not-possible')` case:
```
E [04/Apr/2016:12:19:36 -0400] [Client 14] Returning IPP client-error-not-possible for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-printer1) from localhost
E [04/Apr/2016:12:19:36 -0400] [Client 14] Returning IPP client-error-not-possible for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-printer2) from localhost
E [04/Apr/2016:12:19:36 -0400] [Client 14] Returning IPP client-error-not-possible for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-Enreg%5Fdans%5FDrive) from localhost
E [04/Apr/2016:12:19:36 -0400] [Client 14] Returning IPP client-error-not-possible for CUPS-Add-Modify-Printer (ipp://localhost/printers/GCP-printer3) from localhost
```

Any idea? :)

**Comment by emnik** [comment]
This is also what I am getting from 2 PCs (although trying to add the same google account and printers):
`
root@tutors-office:/home/tutor/Apps/CUPS-Cloud-Print-master# ./upgrade.py 
Fetching list of available ppds...
List retrieved successfully
Updating GCP-__Google_Drive
lpadmin: Unable to copy PPD file.

root@tutors-office:/home/tutor/Apps/CUPS-Cloud-Print-master# cat /var/log/cups/error_log
E [29/Mar/2016:17:10:11 +0300] copy_model: empty PPD file
E [29/Mar/2016:17:10:11 +0300] [Client 20] Returning IPP server-error-internal-error for CUPS-Add-Modify-Printer (ipp://localhost:631/printers/GCP-__Google_Drive) from localhost

root@tutors-office:/home/tutor/Apps/CUPS-Cloud-Print-master# ./setupcloudprint.py 
You currently have these accounts configured: 
nikiforakis.m@gmail.com
Error adding: GCP-EPSON-Epson-Stylus-SX525WD
(1280, u'server-error-internal-error')
Error adding: GCP-EPSON-Epson-Stylus-SX525WD 
Error adding: GCP-Panasonic_DP-8016P_Ver_2
(1280, u'server-error-internal-error')
Error adding: GCP-Panasonic_DP-8016P_Ver_2 
Added 2 new printers to CUPS
`
it says added but they are not,...

**Comment by fallenpixel** [comment]
I am experiencing this issue as well.

**Comment by simoncadman** [comment]
I've released a new version of CUPS Cloud Print ( 20160502 ), should all be working again with this release.

Thanks
